pkg/stanza/operator/helper/multiline.go

```diff
@@ -44,12 +44,12 @@ type MultilineConfig struct {
 }
 
 // Build will build a Multiline operator.
-func (c MultilineConfig) Build(enc encoding.Encoding, flushAtEOF bool, force *Flusher, maxLogSize int) (bufio.SplitFunc, error) {
-	return c.getSplitFunc(enc, flushAtEOF, force, maxLogSize)
+func (c MultilineConfig) Build(enc encoding.Encoding, flushAtEOF bool, force *Flusher, maxLogSize int, preserveWhitespace bool) (bufio.SplitFunc, error) {
```

Comment:
nit: These flags are used in similar ways. Let's keep them together.
```suggestion
func (c MultilineConfig) Build(enc encoding.Encoding, flushAtEOF, preserveWhitespace bool, force *Flusher, maxLogSize int) (bufio.SplitFunc, error) {
```